src/widgets/slider/lv_slider.c

```diff
@@ -49,6 +49,52 @@ static void update_knob_pos(lv_obj_t * obj, bool check_drag);
 /**********************
  *  STATIC VARIABLES
  **********************/
+
+#if LV_USE_OBJ_PROPERTY
+static const lv_property_ops_t properties[] = {
+    {
+        .id = LV_PROPERTY_SLIDER_VALUE,
+        .setter = lv_slider_set_value,
+        .getter = lv_slider_get_value,
+    },
+    {
+        .id = LV_PROPERTY_SLIDER_LEFT_VALUE,
+        .setter = lv_slider_set_left_value,
+        .getter = lv_slider_get_left_value,
+    },
+    {
+        .id = LV_PROPERTY_SLIDER_RANGE,
+        .setter = lv_slider_set_range,
+        .getter = NULL,
+    },
+    {
+        .id = LV_PROPERTY_SLIDER_MIN_VALUE,
+        .setter = NULL,
+        .getter = lv_slider_get_min_value,
+    },
+    {
+        .id = LV_PROPERTY_SLIDER_MAX_VALUE,
+        .setter = NULL,
+        .getter = lv_slider_get_max_value,
+    },
```

Comment:
Please do note that `RANGE` property supports two values for min and max, but the getter does not support. This matches with the C code that has to use MIN_VALUE, and MAX_VALUE.